python/src/robyn/modeling/clusters/cluster_util.py

```diff
@@ -0,0 +1,33 @@
+# pyre-strict
+
+import numpy as np
+import pandas as pd
+from typing import List, Dict, Any
+
+
+def check_opts(value: Any, options: List[Any]) -> None:
+    """Check if a value is in a list of options"""
+    if value not in options:
+        raise ValueError(f"Value {value} not in options: {options}")
+
+
+def removenacols(df: pd.DataFrame, all: bool = False) -> pd.DataFrame:
+    """Remove columns with NA values"""
+    if all:
+        return df.dropna(axis=1, how="all")
+    else:
+        return df.dropna(axis=1, how="any")
+
+
+def formatNum(num: float, abbr: bool = False) -> str:
+    """Format numbers for display"""
+    # Implementation here
+    pass
+
+
+def glued(template: str, **kwargs: Any) -> str:
```

Comment:
What does this method do? Do we need it python?

Reply:
In fact, I would not add any of these methods and let's handle them during implementation.